aws-samples/eks-workshop
# Adding spinnaker chapter

*Issue #, if available:*

*Description of changes:*
In this workshop, we will focus on Application Deployment using Spinnaker:

- How to install Spinnaker and configure Spinnaker services using Kubernetes Operator in Amazon EKS
- Build a simple Spinnaker CD pipeline
   - Stage - Add Ngnix deployment artifact manifest
   - Testing -
    - Manually trigger the pipeline
    - Test the deployment
- Build Helm-based Spinnaker CD Pipeline
   - Configuration
   - Trigger: Continuous Delivery will be triggered automatically based on new image upload into ECR
     - Stage 1 - Bake the manifest from GitHub repo for deployment using Helm
     - Stage 2 - Deploy the Helm artifact to EKS
   - Testing -
      - Upload container image to ECR to trigger the deployment pipeline
       - Test the deployment

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
